core/spec/lib/i18n_spec.rb

```diff
@@ -29,25 +29,35 @@
     I18n.reload!
   end
 
-  it "translates within the spree scope" do
-    expect(Spree.t(:foo)).to eql("bar")
-    expect(Spree.translate(:foo)).to eql("bar")
-  end
+  describe '#translate' do
+    after do
+      Spree::Deprecation.behavior = :raise
```

Comment:
Another approach could be stubbing the deprecation warning 

```
expect(Spree::Deprecation).to receive(:warn)
```

Reply:
Leaving it like that means that from the moment we ran this spec we always raise for deprecations even if we change this in the spec helper in the future. Either wrap the spec in a

```rb
around do |example|
  Spree::Deprecation.silence { example.run }
end
```

or do what @kennyadsl proposes.

Third possibility is to remove specs for deprecated methods.